composer.json

```diff
@@ -14,6 +14,7 @@
 		"php": ">=5.2"
 	},
 	"require-dev": {
+        "phpunit/phpunit": ">=4.0",
```

Comment:
Need to check the minimal version providing the "forword compat" stuff

AFAIK: ^4.8.35 || ^5.7 || ^6.0

Reply:
Indeed, 4.8.35, but this is not needeed since composer checks out the latest version compatible with the current PHP version it is testing and 4.8.35 works on PHP 5.3+ (yes, this will screw PHP 5.2)